src/platform-aws.c

```diff
@@ -26,34 +26,40 @@ struct ec2_platform_data {
 	const char* topology;
 	int default_dup_conns;
 	float latency;
+	bool gdr_supported;
```

Comment:
Consider naming this `gdr_required` to express the intent.  This name leaves some confusion on whether or not the flag must be true for gdr to work, which isn't what the code does.